pkg/util/log/logger.go

```diff
@@ -54,10 +55,10 @@ func New(opts ...Option) (*zap.Logger, error) {
 	core := zapcore.NewCore(encoder, syncer, level)
 
 	// Recommended zap.Option
-	zapOpts := append(cfg.zapOpts,
+	zapOpts := slices.Concat([]zap.Option{
```

Comment:
What does the change mean?

Reply:
As the title says, it avoids overriding the given options with hard-coded recommended options, so it can keep the user-provided options.